testdata/sqllogictests/functions/kdl.slt

```diff
@@ -36,12 +37,12 @@ select count(*) from cows where kdl_matches(cows.docs, '[status]');
 ----
 2
 
-query
+query I
 select cows.docs from cows where kdl_matches(cows.docs, '[b=100]');
 ----
 name "betsy" status="amazing" age=89; details "classic" a=1 b=100;
 
-query
+query I
```

Comment:
```suggestion
query T
```

Reply:
are there any docs on what the different `query <VALUE>` arguments are? I've always just copypasta'd other examples.

Reply:
> The characters codes are "T" for a text result, "I" for an integer result, and "R" for a floating-point result. 

From https://www.sqlite.org/sqllogictest/doc/trunk/about.wiki

We had in the contributing doc at point, but seems like it was removed.